_includes/image.html

```diff
@@ -9,8 +9,11 @@
 
   @param {string} "file" - Required. filename of the image.
   @param {string} "alt" - Required. Alt attribute for image tag.
-  @param {boolean} "img_link" - Optional, default is false. If img should have
-    the image overlay functionality that the theme provides.
+  @param {string} "img_link" - Optional. The link when an image is clicked.
+    Possible values for img_link are:
+    - default is false or null. Means no link for the image.
+    - true. Means image will have overlay functionality that theme provides.
+    - an actual url.  Provides a page link to go to when image is clicked.
```

Comment:
Can we make a separate parameter for URLs, like `link_url`? It feels a bit untidy to have overload `img_link` to where it can be null, true, false, or a string.

So it would be like:

```html
{% include image.html file="arduino-starter-kit.jpg" alt="Arduino starter kit" max_width="300px" class="align-left" link_url="http://amzn.to/2rpRLCf" %}
```

Reply:
Absolutely, I agree with you, that's a good point.  I changed the logic to look first for the `link_url` param and then the `img_url` param.  So, the possibilities are:

| param(s) included | rendering result |
| ------------------ | ---------------- |
| `link_url` | image will render with the link url value as the href in the `<a>` tag around the image |
| `img_url` | image will render with the image overlay capability |
| both `img_url` and `link_url` | image will use the `link_url` and  render with the link url value as the href in the `<a>` tag around the image |
| neither `img_url` or `link_url` | no link for the image |

This update is completed in 4903898